### PR TITLE
Avoid force unwrapping in the Swift examples

### PR DESCRIPTION
**Pull request by jpsim**
We shouldn't be recommending that users do this.

The times in which values must not be nil, I'm using asserts instead. /cc @segiddins @tgoyne

### CODE REVIEW

**Comment by segiddins** [comment]
:+1:

**Comment by tgoyne** [comment]
This seems pointless. Asserting before force unwrapping doesn't give any extra safety, and doesn't even improve the error in debug builds much. If we want to have our sample code handle nils in places where they shouldn't be possible, then it needs to actually do that.

**Comment by tgoyne** [comment]
Well, most of the changes are good. Just the assert-then-force-unwrap pattern is pointless.

**Comment by jpsim** [comment]
Alright, I'll revert those. Sam had the same comment.

File: examples/ios/swift/Encryption/ViewController.swift [hunk 1]
```diff
@@ -78,26 +74,28 @@ class ViewController: UIViewController {
...
     func log(text: String) {
-        self.textView.text = self.textView.text + text + "\n\n"
+        textView.text = textView.text + text + "\n\n"
     }
 
     func getKey() -> NSData {
         // Identifier for our keychain entry - should be unique for your application
         let keychainIdentifier = "io.Realm.EncryptionExampleKey"
-        let keychainIdentifierData = keychainIdentifier.dataUsingEncoding(NSUTF8StringEncoding, allowLossyConversion: false)
+        let optionalKeychainIdentifierData = keychainIdentifier.dataUsingEncoding(NSUTF8StringEncoding, allowLossyConversion: false)!
```
Comment (tgoyne) [inline examples/ios/swift/Encryption/ViewController.swift]: Rename needs to be reverted too?

Comment (jpsim) [inline examples/ios/swift/Encryption/ViewController.swift]: How right you are! Done.

**Pull request merged**